jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/jpa/hibernate/PostgreSQLByteaTypeContributor.java

```diff
@@ -39,11 +41,22 @@ public ByteaContributorType() {
 
     }
 
+    public class TextContributorType extends StandardBasicTypeTemplate<String> {
+
+        private static final long serialVersionUID = 1619875355308645967L;
+
+        public TextContributorType() {
+            super(LongVarcharTypeDescriptor.INSTANCE, StringTypeDescriptor.INSTANCE, StandardBasicTypes.MATERIALIZED_CLOB.getName());
+        }
+
+    }
+
     @Override
     public void contribute(TypeContributions typeContributions, ServiceRegistry serviceRegistry) {
         final Dialect dialect = serviceRegistry.getService(JdbcServices.class).getDialect();
         if (dialect instanceof org.hibernate.dialect.PostgreSQL81Dialect && Boolean.getBoolean("org.kie.persistence.postgresql.useBytea")) {
             typeContributions.contributeType(new ByteaContributorType());
+            typeContributions.contributeType(new TextContributorType());
```

Comment:
Please add a system property to enable the TextContributorType for backwards compatibility when running against an existing database. By default, it should not be enabled.